testsuite/unstable-api-annotation/feature-pack/subsystem/src/main/java/org/wildfly/core/test/unstable/api/annotation/classes/_private/UnstableAnnotationApiTestLogger.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  JBoss, Home of Professional Open Source.
+ *  Copyright 2024 Red Hat, Inc., and individual contributors
+ *  as indicated by the @author tags.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
```

Comment:
If this is really new code we want to use the updated license header in Java files:

```
/*
 * Copyright The WildFly Authors
 * SPDX-License-Identifier: Apache-2.0
 */
```